ash/src/lib.rs

```diff
@@ -91,6 +91,19 @@ impl<'r, T> RawPtr<T> for Option<&'r T> {
     }
 }
 
+pub trait RawMutPtr<T> {
+    unsafe fn as_raw_mut_ptr(self) -> *mut T;
+}
+
+impl<'r, T> RawMutPtr<T> for Option<&'r mut T> {
+    unsafe fn as_raw_mut_ptr(self) -> *mut T {
+        match self {
+            Some(val) => val,
+            None => core::ptr::null_mut(),
```

Comment:
Let's make the above match arm `None =>` too instead of `_ =>`, and match the `::core` -> `core` convention too? We barely use that anywhere outside of generated code.